tests/profiles/layout_test.py

```diff
@@ -5,13 +5,13 @@
     CheckTester,
     TEST_FILE,
 )
-from fontbakery.profiles import layout as layout_profile
+from fontbakery.profiles import googlefonts as googlefonts_profile
 
 
 def test_check_layout_valid_feature_tags():
     """Does the font have any invalid feature tags?"""
     check = CheckTester(
-        layout_profile, "com.google.fonts/check/layout_valid_feature_tags"
+        googlefonts_profile, "com.google.fonts/check/layout_valid_feature_tags"
```

Comment:
why?

Reply:
I'm not sure we want a separate layout profile.

Reply:
I agree.